samples/algorithms/sudoku-grover/ColoringGroverWithConstraints.qs

```diff
@@ -60,23 +60,20 @@ namespace Microsoft.Quantum.Samples.ColoringGroverWithConstraints {
     }
 
     /// # Summary
-    /// Oracle for verifying vertex coloring, including color constraints from
-    /// non qubit vertices.
+    /// Oracle for verifying vertex coloring
     ///
     /// # Input
-    /// ## numVertices
+    /// ## nVertices
     /// The number of vertices in the graph.
     /// ## bitsPerColor
     /// The bits per color e.g. 2 bits per color allows for 4 colors.
     /// ## edges
     /// The array of (Vertex#,Vertex#) specifying the Vertices that can not be
     /// the same color.
-    /// ## startingColorConstraints
-    /// The array of (Vertex#,Color) specifying the disallowed colors for vertices.
     ///
     /// # Output
-    /// A unitary operation that applies `oracle` on the target register if the control
-    /// register state corresponds to the bit mask `bits`.
+    /// An marking oracle that with the purpose of marking states as allowed where the colors of qubits related by an edge constraints are not equal.
```

Comment:
I'm not sure this is gramatically correct? "That with" feels awkward at the least.

```suggestion
    /// An marking oracle that marks as allowed those states in which the colors of qubits related by an edge constraint are not equal.
```